File: app/src/main/java/com/ja/assets/ui/activity/other/SweepCodeActivity.java
package com.ja.assets.ui.activity.other;

import android.widget.TextView;

import androidx.appcompat.widget.LinearLayoutCompat;

import com.ja.assets.R;
import com.ja.assets.databinding.ActivitySweepCodeBinding;
import com.ja.assets.ui.base.BaseActivity;
import com.ja.assets.utils.ToastUtil;

public class SweepCodeActivity extends BaseActivity {

    private ActivitySweepCodeBinding sweepCodeBinding;
    private String traceCode;


    @Override
    protected int getLayoutId() {
        return R.layout.activity_sweep_code;
    }

    @Override
    protected void initView() {
        TextView titleCenterText = findViewById(R.id.titleCenterText);
        titleCenterText.setText(ToastUtil.getString(R.style.assetsInfo));

        LinearLayoutCompat commonLeftLinearLayout = findViewById(R.id.commonLeftLinearLayout);
        sweepCodeBinding = (ActivitySweepCodeBinding) getViewDataBinding();
        commonLeftLinearLayout.setOnClickListener(v -> finish());
        traceCode = getIntent().getStringExtra("traceCode");
    }

    @Override
    protected void initAdapter() {

    }

    @Override
    protected void initData() {

    }
}

File: app/src/main/java/com/ja/assets/ui/activity/other/SweepCodeActivity.kt
package com.ja.assets.ui.activity.other

import com.ja.assets.R
import com.ja.assets.databinding.ActivitySweepCodeBinding
import com.ja.assets.ui.base.BaseActivity
import com.ja.assets.utils.ToastUtil
import kotlinx.android.synthetic.main.common_title.*

class SweepCodeActivity : BaseActivity() {

    private var sweepCodeBinding:ActivitySweepCodeBinding?=null
    private var traceCode:String=""
    override fun getLayoutId(): Int {
        return R.layout.activity_sweep_code
    }

    override fun initView() {
        titleCenterText.text=ToastUtil.getString(R.style.assetsInfo)
        commonLeftLinearLayout.setOnClickListener { finish() }
        sweepCodeBinding=viewDataBinding as ActivitySweepCodeBinding
        traceCode=intent.getStringExtra("traceCode")!!
    }

    override fun initAdapter() =Unit

    override fun initData() {
        //获取资产详情

    }
}
